pkg/block/namespace.go

```diff
@@ -126,7 +126,7 @@ func ResolveNamespace(defaultNamespace, key string, identifierType IdentifierTyp
 func resolveFull(key string) (QualifiedKey, error) {
 	parsedKey, err := url.ParseRequestURI(key)
 	if err != nil {
-		return QualifiedKey{}, fmt.Errorf("key isn't a valid address: %w", err)
+		return QualifiedKey{}, fmt.Errorf("Could not parse URI: %w", err)
```

Comment:
Generally the negative is already in the base error message, so it may make sense merely to describe the failure:
```suggestion
		return QualifiedKey{}, fmt.Errorf("parse URI: %w", err)
```